iothub/device/src/Transport/Mqtt/MqttTransportHandler.cs

```diff
@@ -962,15 +974,12 @@ public override async Task SendTwinPatchAsync(TwinCollection reportedProperties,
                 var websocket = new ClientWebSocket();
                 websocket.Options.AddSubProtocol(WebSocketConstants.SubProtocols.Mqtt);
 
-                // Check if we're configured to use a proxy server
-                IWebProxy webProxy = settings.Proxy;
-
                 try
                 {
-                    if (webProxy != DefaultWebProxySettings.Instance)
+                    if (isProxyConfigured())
```

Comment:
C# case would be IsProxyConfigured

Reply:
I guess I woke up on the Java side of my bed this morning, oops